app/lib/importers/file_importer.rb

```diff
@@ -16,7 +16,7 @@ def import
     CSV.foreach(csv_filespec || [], headers: true, header_converters: :symbol) do |row|
       yield(row)
       @number_imported += 1
-    rescue StandardError => e
```

Comment:
interesting, I wonder why it did this

Reply:
looks like an unused argument issue. If e is never used, it's better left out